manifests/server/grant_role.pp

```diff
@@ -0,0 +1,43 @@
+# Define for granting membership to a role. See README.md for more information
+define postgresql::server::grant_role (
+  $group,
+  $role,
+  $ensure           = 'present',
+  $psql_db          = $postgresql::server::default_database,
+  $psql_user        = $postgresql::server::user,
+  $port             = $postgresql::server::port,
+  $connect_settings = $postgresql::server::default_connect_settings,
+) {
+  case $ensure {
+    'present': {
+      $command = "GRANT '${group}' TO '${role}'"
```

Comment:
Hi there, just got a note from Lindsey to look into this PR. Thanks for contributing! It looks like there are still some open issues that @bmjen raised around validation. Could you address those in the PR?